### PR TITLE
[cli][docs][www] add unistyles template

### PR DESCRIPTION
**Pull request by coyksdev**
## Description

This PR added the [react-native-unistyles](https://reactnativeunistyles.vercel.app/) styling template, offering a convenient solution for cross-platform styling in React Native. Additionally, it includes updates to the documentation and the landing page on the website.

## How Has This Been Tested?
- generate each packages with --unistyles option
- add unit test for unistyles
- check and run the website's landing page
- check and run the website's documentation  

### Landing page:

Screenshot 2024-01-23 at 11 11 40 pm

### Sample generated app:

Screenshot 2024-01-23 at 11 09 43 pm

### CODE REVIEW

**Comment by danstepanov** [comment]
@coyksdev I just merged in a PR that added Restyle, can you rebase with `main`? Perhaps @ludwig-pro might be down to help?

**Comment by coyksdev** [comment]
rebase done @danstepanov

**Review by ludwig-pro — APPROVED** [review]
Great PR! I'm excited to start using unistyles! It's a pity that you didn't use the theme in components with createStyleSheet. Do you think it would be worthwhile to do it?

```
const stylesheet = createStyleSheet(theme => ({
    container: {
        flex: 1,
        justifyContent: 'center',
        alignItems: 'center'
        backgroundColor: theme.colors.secondary // use the theme in the template files ?
    },
    }))
```

File: cli/src/utilities/runCLI.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ export async function runCLI(toolbox: Toolbox, projectName: string): Promise<Cli
     type: 'select',
     name: 'stylingSelect',
     message: 'What would you like to use for styling?',
-    choices: ['Nativewind', 'Restyle', 'Stylesheet', 'Tamagui (experimental)']
+    choices: ['Nativewind', 'Restyle', 'Stylesheet', 'Tamagui (experimental)', 'Unistyles']
```
Comment (danstepanov) [inline cli/src/utilities/runCLI.ts:76]:
```suggestion
    choices: ['Stylesheet', 'Nativewind', 'Restyle',  'Unistyles', 'Tamagui (experimental)']
```

**Review by danstepanov — COMMENTED** [review]
Per @ludwig-pro's comment, would love to see a more theme-based approach to styles here. It would really help us reduce the amount of style code in the repo. If that is too large of a commitment then we can punt that to another PR. lmk

**Pull request merged**